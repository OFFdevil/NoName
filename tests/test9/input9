def FUNCa A
{
    // body
}

def FUNCa (struct A)
{
    // body
}

main
  FUNCa( C),FUNCa( (struct D));
    FUNCa( A);
    FUNCa( C);
     FUNCa( B);
  FUNCa( F),FUNCa( E);
    FUNCa( A);
     FUNCa( E);
      FUNCa( Z);
 FUNCc( B);
 FUNCa( C);