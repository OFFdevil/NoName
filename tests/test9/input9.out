2
4
4
5
2
4
5
6
1
1
3запускаем отношение в строке с количеством пробелов =  4
3запускаем отношение в строке с количеством пробелов =  5
3запускаем отношение в строке с количеством пробелов =  4
3запускаем отношение в строке с количеством пробелов =  2
3запускаем отношение в строке с количеством пробелов =  6
3запускаем отношение в строке с количеством пробелов =  5
3запускаем отношение в строке с количеством пробелов =  4
3запускаем отношение в строке с количеством пробелов =  2
3запускаем отношение в строке с количеством пробелов =  1
3запускаем отношение в строке с количеством пробелов =  1
